### ISSUE TITLE
Customize similarity of topics reduction

### ISSUE THREAD

**Issue by paulthemagno** [MaartenGr/BERTopic#330]
Hi! I saw there is this parameter: `nr_topics` with this description:

```
Specifying the number of topics will reduce the initial number of topics to the value specified.
This reduction can take a while as each reduction in topics (-1) activates a c-TF-IDF calculation.
If this is set to None, no reduction is applied. Use "auto" to automatically reduce topics that have a similarity of at least 0.9, do not maps all others.
```

How can I change the percentage of similarity? Example: merge together similar topics at 0.8.

I see that for example with `nr_topics= "auto"` the class calls this function: [_auto_reduce_topics](https://github.com/MaartenGr/BERTopic/blob/15ea0cd804d35c1f11c6692f33c3666b648dd6c8/bertopic/_bertopic.py#L1727) but I don't see where `0.9` is put as percentage of similarity.

Thank you!

**Comment by MaartenGr** [comment]
Good catch! It seems that the description is currently outdated as the "auto" step does not use that similarity value anymore. Whenever you use `nr_topics="auto"`, a fresh HDBSCAN instantiation is used to cluster all topics automatically. After the topics are clustered, new topic representations are generated. 

I have put this on the to-do list and will most likely be fixed in the new release!

**Comment by paulthemagno** [comment]
Thanks, @MaartenGr. In your opinion, at the moment, which are the best parameters to pass to bertopic to have the best topics reduction?

**Comment by MaartenGr** [comment]
That very much depends on what you define as "best topics". If, for example, you know there are a certain amount of topics in the documents and not much more than that, setting `nr_topics` to a specific integer would be preferred. However, setting it to "auto" does cluster the topics more naturally. I cannot say which one is better than the other but starting out with "auto" typically gives you a good understanding of the number topics, in their reduced state, in the documents.

**Comment by MaartenGr** [comment]
This was fixed in the v0.9.4 release of BERTopic, so I'll be closing this issue for now. Let me know if you run into any other issues!
